src/main/kotlin/io/github/mojira/arisa/infrastructure/HelperMessages.kt

```diff
@@ -115,7 +115,9 @@ data class HelperMessages(
         fun deserialize(json: String) = Klaxon().parse<HelperMessages>(json)
 
         fun fetch() = with(URL(url).openConnection() as URLConnection) {
-            deserialize(content as String).rightIfNotNull { Error("Couldn't download or deserialize helper messages") }
+            Klaxon().parse<HelperMessages>(inputStream).rightIfNotNull {
```

Comment:
Where does the variable `inputStream` come from?

Reply:
he is inside a with, so from the UrlConnection

Reply:
It's a property of `URLConnection`

Reply:
Oh, the with, it's probably a field. merging.